cmd/finch/support_bundle.go

```diff
@@ -60,7 +60,7 @@ func newGenerateSupportBundleAction(
 	}
 }
 
-func (gsa *generateSupportBundleAction) runAdapter(cmd *cobra.Command, args []string) error {
+func (gsa *generateSupportBundleAction) runAdapter(cmd *cobra.Command, cargs []string) error {
```

Comment:
No need this change?

Reply:
this is not my change, I will take a look why it is here